app/services/support_interface/ministerial_report_candidates_export.rb

```diff
@@ -15,26 +15,43 @@ def call(*)
 
       export_rows[:split] = column_names
 
-      application_forms.find_each do |application|
-        next if application.phase == 'apply_2'
+      subject_report = {}
 
+      application_forms.find_each do |application|
         subjects = determine_subjects(application)
```

Comment:
This seems to be always working out the (dominant) subject from the _apply_1_ application (in cases where the candidate has multiple applications). Should it not be doing the same thing as we do below for determining states, i.e. find the latest viable application?

Reply:
good q — one for Andrew, I think?